WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -3,31 +3,37 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="title" content="Wasabi Wallet - Unfairly Private">
+    <meta name="title" content="Wasabi Wallet - Unfairly Private Bitcoin Wallet">
     <meta name="description" content="Open-source, non-custodial, privacy-focused Bitcoin wallet, that implements trustless coin shuffling with mathematically provable anonymity - Chaumian CoinJoin.">
 
     <!-- Open Graph / Facebook -->
     <meta property="og:type" content="website">
     <meta property="og:url" content="https://www.wasabiwallet.io">
-    <meta property="og:title" content="Wasabi Wallet - Unfairly Private">
+    <meta property="og:title" content="Wasabi Wallet - Unfairly Private Bitcoin Wallet">
     <meta property="og:description" content="Open-source, non-custodial, privacy-focused Bitcoin wallet, that implements trustless coin shuffling with mathematically provable anonymity - Chaumian CoinJoin.">
     <meta property="og:image" content="https://www.wasabiwallet.io/images/wasabi-social.jpg">
 
     <!-- Twitter -->
     <meta property="twitter:card" content="summary_large_image">
     <meta property="twitter:url" content="https://www.wasabiwallet.io">
-    <meta property="twitter:title" content="Wasabi Wallet - Unfairly Private">
+    <meta property="twitter:title" content="Wasabi Wallet - Unfairly Private Bitcoin Wallet">
     <meta property="twitter:description" content="Open-source, non-custodial, privacy-focused Bitcoin wallet, that implements trustless coin shuffling with mathematically provable anonymity - Chaumian CoinJoin.">
     <meta property="twitter:image" content="https://www.wasabiwallet.io/images/wasabi-social.jpg">
 
     <link href="https://fonts.googleapis.com/css?family=Inconsolata:400,700|Playfair+Display:700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="css/bootstrap.css?ver=1.1.0">
     <link rel="stylesheet" href="css/style.css?ver=1.1.0">
     <link rel="shortcut icon" href="images/favicon.ico" />
-    <title>Wasabi Wallet - Unfairly Private</title>
+    <title>Wasabi Wallet - Unfairly Private Bitcoin Wallet</title>
 </head>
 <body>
-
+<div itemscope itemtype="https://schema.org/SoftwareApplication">
+      <link itemprop="applicationCategory" href="https://schema.org/FinanceApplication"/>
+<div itemprop="offers" itemscope itemtype="http://schema.org/Offer">
+  <meta itemprop="name" content="Wasabi Wallet">
+  <meta itemprop="price" content="0">
+  <meta itemprop="priceCurrency" content="BTC">
+</div>
```

Comment:
This is not visible, it's just meta data right? Not even the `link` tag?

Reply:
Exactly.